### ISSUE TITLE
Bunch of Spf13 issues!!

### ISSUE THREAD

**Issue by entrptaher** [spf13/spf13-vim#688]
I admit spf13.vim is the quickest, but when it doesn't work, it doesn't work really hard. Here are the issues I came up with.
- I cannot install it in windows 8.1, it just stuck at the end.
- I cannot install it using cinst either.
- If I accidentally open the .cmd, it just starts everything from beginning, even if everything is present.
- Not to mention gVim becomes slower (like Atom) if spf13 is successfully installed.

I wonder if the development for windows is stopped.

**Comment by ghost** [comment]
the continuing development and usage on windows requires folks who are able to deal with windows issues. I know i have no experience in that.  I wonder what windows versions @spf13 deals with on the day to day. Maybe it's just windows 7.

**Comment by spf13** [comment]
Only Windows 7 personally

We really need maintainers to step up and shepherd this who use Windows daily for dev. I'm happy to give commit access to anyone who wants to volunteer.

**Comment by entrptaher** [comment]
At last it worked.

I'll describe what happened, what worked and what didn't,

Previously I tried on...
Intel processor, Windows 7, 2 GB Ram, Intel HD Graphics
Amd processor, Windows 8.1 (32 bit and 64 bit), (2 and 4) GB Ram, 1 Gb ATI graphics,

Now, I tried to install it in...
Intel processor, windows 8.1 64bit, 4GB RAM, 2GB NVidia Graphics...

I installed Git, Curl, Vim normally... Even if I did that, Choco installed them again (What a pain!!!)... It didn't matter which way I installed them though...
1. Installing Spf13-vim via Choco...FAILURE..., 
   ![](http://i.imgur.com/X26SzPO.jpg)
2. I tried to uninstall and reinstall again via Choco...FAILURE...
   Showed "Not a git repo" alongside other problems...
   ![](http://i.imgur.com/4fPiBJR.jpg)
3. Installed using .CMD file AS ADMINISTRATOR...Took a while...didn't get stuck like previous time...
   SUCCESS...!!
   ![](http://i.imgur.com/54wEcEC.jpg)

NOTE: previously, I did the same and stuck at some point...
NOTE 2: Now, it ran way much faster, looks fun! Maybe it depends on Hardware/architecture when it comes to Windows....

I'll try to notify if any other problem occurs/ something comes up in mind in future...
